skynet.py

```diff
@@ -0,0 +1,43 @@
+import os
```

Comment:
Awesome, thanks for doing this! Might be worth moving this into a custom skynet-images branch so that we can use this approach in other repos.